### PR TITLE
Add get_state_string()

### PR DESCRIPTION
**Pull request by manuels**
Pretty useful for debugging ;)

### CODE REVIEW

File: openssl/src/ssl/mod.rs [hunk 1]
```diff
@@ -690,6 +691,24 @@ impl Ssl {
         Ok(ssl)
     }
 
+    pub fn get_state_string<'a>(&self) -> &'a str {
+        let state = unsafe {
```
Comment (sfackler) [inline openssl/src/ssl/mod.rs]: What is the lifetime of the strings returned by these methods? If it's static, returning a `&'static str` is a bit more clear than an unbound lifetime.

Comment (manuels) [inline openssl/src/ssl/mod.rs]: The lifetime isn't static, so an unbound lifetiem should be appropriate:

    const char *SSL_state_string(const SSL *ssl);
    const char *SSL_state_string_long(const SSL *ssl);

Comment (sfackler) [inline openssl/src/ssl/mod.rs]: An unbounded lifetime is equivalent to static: http://is.gd/xjEcoa

**Comment by manuels** [comment]
Sorry, I fix the issue with the static lifetime.

**Pull request merged**